vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/MouseActions.java

```diff
@@ -200,14 +198,7 @@ public void moveToElement(WebElement element)
             {
                 javascriptActions.scrollIntoView(element, true);
             }
-            try
-            {
-                new Actions(getWebDriver()).moveToElement(element).perform();
-            }
-            catch (MoveTargetOutOfBoundsException ex)
-            {
-                softAssert.recordFailedAssertion(COULD_NOT_MOVE_ERROR_MESSAGE + ex);
-            }
+            new Actions(getWebDriver()).moveToElement(element).perform();
```

Comment:
Why handling removed?

Reply:
According to Javadocs it shouldn't be thrown